### ISSUE TITLE
NTR: N-acetylaspartate absent in CSF (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8637]
New term request N-acetylaspartate absent in CSF (segal_170306192708): NAA; cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000674
- label: Decreased CSF N-acetylaspartic acid concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: 
- def: Concentration of N-acetylaspartic acid in the cerebrospinal fluid below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8637
- synonyms: Decreased CSF N-acetylaspartate acid concentration(NONE)|N-acetylaspartate absent in CSF(NONE)
